src/SupportMessage/SupportMessage.tsx

```diff
@@ -0,0 +1,100 @@
+import React, { FC } from 'react'
+import styled, { css } from 'styled-components'
+import { lighten } from 'polished'
+
+import { Box } from '../Box'
+import { Icon } from '../Icon'
+import { theme } from '../theme'
+
+export type SupportMessageProps = {
+  className?: string
+  children: string
+  type: 'info' | 'alert' | 'warning'
+  title?: string
+  hasBackground?: boolean
+  hasBorder?: boolean
+  borderColor?: string
```

Comment:
Those seems too low level in term of customisation
I think we need to clarify a few points with design before finding a good solution on this part

Reply:
you were thinking to have an extra type for the bordered one instead?

Reply:
This could be a potential solution yes
Not sure about the best way to go but it would be better as it would pair the background, border and border color together instead of having it separated like that

Reply:
To pair everything together we need to rely on the whole state prop and just style it according to it. I think it's easily doable and it's definitely less customisable and more close to Figma designs.
The only problem is that design will have to change the info state cause currently has both a bordered and non bordered version